src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -370,4 +375,32 @@ private void changeLabels() {
             ui.triggerEvent(new ShowLabelPickerEvent(getSelectedIssue().get()));
         }
     }
+
+    /**
+     * Triggered when a RepoOpeningEvent is received by the filter panel.
```

Comment:
A method has not control over when it is called. So the method comment should explain only what the method does.